README.md

```diff
@@ -3,26 +3,130 @@
 
 <img src='https://raw.githubusercontent.com/vince-fugnitto/local-history-ext/master/assets/local-history.png' width='125px'/>
 
-</div>
-
 # Local History
 
+
+</div>
+
 <img src='https://api.travis-ci.com/vince-fugnitto/local-history-ext.svg?branch=master' />
 
-### Description
 
-The **Local History** extension manages local revisions of a file independently of source control providers, and allows users to easily view, track, and restore files based on previous checkpoints. The extension is particularly useful for disaster recovery.
+## Description
+
+The **Local History** extension manages local revisions of a file independently of source control providers, and allows users to easily view, track and restore files based on previous saves. The extension automatically creates save backups for the user so they are able to easily recover past revisions which is especially useful for disaster recovery.
+
+## Features
+
+The following are some high-level features the extension provides:
+
+- ### Automatic Revisions
+
+  The extension creates automatic revisions for the user based on when files are saved in their workspace. These automatic revisions are tracked by the extension and users are capable or reviewing them directly in the application. 
+
+- ### View Revisions as a `Diff`
+
+  When reviewing a revision for an active editor, the revision will be displayed as a `diff` (lines added versus deleted) so that users are able to easily view the differences between the revision and the content of the current editor.
```

Comment:
When reviewing a revision for an active editor, the `read-only` revision will be displayed as